buildsystem/modules/FindPython.cmake

```diff
@@ -116,6 +116,7 @@ else()
 		"/usr/local/Frameworks/Python.framework/Versions/*/bin/python*"
 		"~/Library/Frameworks/Python.framework/Versions/*/bin/python*"
 		"/System/Library/Frameworks/Python.framework/Versions/*/bin/python*"
+		"/usr/local/opt/python/Frameworks/Versions/*/bin/python*"
```

Comment:
That line should be the only thing needed to find python and its libs correctly, I hope.

Reply:
Turns out this is pointless - both `/usr/local/Frameworks/Python.framework/Versions` and `/usr/local/opt/python/Frameworks/Versions` symlink to the same place.